CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Changelog
+---
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased] - yyyy-mm-dd
+
+Just minor documentation fixes.
+
+### Added
+- [`<Tab>`](https://github.com/illright/attractions/blob/master/attractions/tab/tab.svelte)s
+  now also dispatch `click` event.
+
+### Changed
```

Comment:
Also how about we omit unused headings?

Reply:
I thought of keeping them at least in the unreleased section so that we remember what sections there are, but I guess it should be intuitive enough to remember. At worst we can consult https://keepachangelog.com/ again